test/edu-benefits/1990e/config/benefitSelection.unit.spec.jsx

```diff
@@ -16,9 +16,8 @@ describe('Edu 1990e benefitEligibility', () => {
           data={{}}
           uiSchema={uiSchema}/>
     );
-    const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
-
-    expect(fields.length).to.equal(2);
+    expect(ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input'))
```

Comment:
I'm not super happy with testing counts, but I think that's better than testing just the presence of inputs. We don't change input types that often and it's more likely to catch schema issues (which sometimes show up as an error message on the form, in lieu of the right input fields).